pdns/dnsdistdist/dnsdist-web.cc

```diff
@@ -612,6 +612,8 @@ static void handlePrometheus(const YaHTTP::Request& req, YaHTTP::Response& resp)
   output << "# TYPE " << statesbase << "healthcheckfailuresmismatch "     << "counter"                                                                              << "\n";
   output << "# HELP " << statesbase << "healthcheckfailuresinvalid "      << "Number of health check attempts where the DNS response was invalid"                   << "\n";
   output << "# TYPE " << statesbase << "healthcheckfailuresinvalid "      << "counter"                                                                              << "\n";
+  output << "# HELP " << statesbase << "healthchecklatency "              << "Latency of the last successful health check attempt, in milliseconds"                << "\n";
```

Comment:
I wonder if we'd also need one for the last health-check, whether successful or unsuccessful.

Reply:
I'm not sure how useful it would be: if it's a timeout there is no relevant latency, and do we actually care about the latency of a servfail/refused/etc answer?